jsonschema-core.xml

```diff
@@ -2777,22 +2776,20 @@
                         </artwork>
                     </figure>
                     <t>
-                    Note that this pointer may not be resolvable by the normal JSON Pointer process
-                    due to the inclusion of these by-reference applicator keywords.
+                        Note that this pointer may not be resolvable by the normal JSON Pointer process
+                        due to the inclusion of these by-reference applicator keywords.
                     </t>
                     <t>
                         The JSON key for this information is "evaluationPath".
                     </t>
                 </section>
 
-                <section title="Keyword Absolute Location">
+                <section title="Subschema Absolute Location">
                     <t>
-                        The absolute, dereferenced location of the validating keyword.  The value MUST
-                        be expressed as a full IRI using the canonical IRI of the relevant schema resource
-                        with a JSON Pointer fragment, and it MUST NOT include by-reference applicators
-                        such as "$ref" or "$dynamicRef" as non-terminal path components.
-                        It MAY end in such keywords if the error or annotation is for that
-                        keyword, such as an unresolvable reference.
+                        The absolute, dereferenced location of the validating subschema.  The value MUST
+                        be expressed as a full IRI using the canonical IRI of the relevant
+                        schema object, and it MUST NOT include by-reference applicators
```

Comment:
Can we mandate consistent use of JSON Pointer fragments instead?  I believe we changed the wording so that "canonical IRI" is only relevant to whole resources (no fragments).  So really it would be canonical IRI plus JSON Pointer fragment.  If this is too big of a topic I'm happy to take it back out to the larger discussion and let it be handled in a later PR.

Reply:
Yeah, let's take this separately.

Reply:
I think I have a wording that might work for this, but it's still definitely worth discussing externally to this PR.